packages/web-react/src/components/Tabs/README.md

```diff
@@ -0,0 +1,90 @@
+# Tabs
+
+The Tab component is used to organize related content. User can navigate between
+groups of information in tabbable regions.
+
+## Tab
+
+```jsx
+<Tabs defaultSelectedTab={1}>
+  <Tab>
```

Comment:
Maybe `<TabList>` would better describe the purpose? I mean seeing `Tab` inside `Tabs` is OK only until I realize that `Tab` is not an item but just a wrapper for items.

Reply:
Ok, renamed to `TabList`.